src/data/markdown/docs/05 Examples/01 Examples/22 instant-load-increase.md

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Instant load increase'
+excerpt: 'Scripting example on how to instantly increase the number of VUs or iterations and hold them for a period of time'
+draft: 'false'
+---
+
+One of the common usages of the loadtesting tools it's the so-called stepped arrival rate.
+
+In K6 we can achieve it with the following configuration.
```

Comment:
```suggestion
In k6 we can achieve it with the following configuration.
```
we generally use lowercase for the k6 brand

Reply:
I can change this.